app/src/Homepage/Homepage.css

```diff
@@ -15,8 +15,9 @@
 /* Sidebar css */
 .sidebar {
   background-color: #F0F0F0;
```

Comment:
This isn't you but can you change these to lowercase to follow the style guide?

Reply:
Sure